src/Acorn/Console/Commands/Command.php

```diff
@@ -2,25 +2,19 @@
 
 namespace Roots\Acorn\Console\Commands;
 
+use Symfony\Component\Process\Process;
+use Illuminate\Support\Str;
 use Illuminate\Console\Command as CommandBase;
-use Illuminate\Console\Scheduling\Schedule;
 
 abstract class Command extends CommandBase
 {
     /**
-     * Holds an instance of the app.
+     * The application implementation.
      *
      * @var \Roots\Acorn\Application
      */
     protected $app;
 
-    /**
-     * {@inheritdoc}
-     */
-    public function schedule(Schedule $schedule)
-    {
-    }
```

Comment:
Is there a reason we had this in the first place? Why is it being removed?

Reply:
I'm 99% sure it was required initially to extend `Command` but it appears to of [been removed](https://github.com/illuminate/console/blob/master/Command.php) in 6.x although I'm not sure which commit. In any case, everything is fine without it.